woocommerce/woocommerce
# Removed Minified React Admin Assets

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

The unminified assets don't _really_ provide very much value since they undergo so much transpilation in either case. They just add to the archive size and unnecessary complication. Since we are open-source it's very easy for people to see the source and so this PR proposes removing these assets entirely. One other thing to consider is that we're already doing this with Blocks so there's precedent for it.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Run `pnpm --filter='@woocommerce/plugin-woocommerce' build`. This will create a production build and we should make sure that the assets are minified and load correctly.
2. Run `pnpm --filter='@woocommerce-plugin-woocommerce' watch:build`. This creates a development build and we should make sure the assets are minified and load correctly.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Remove Unminified React Admin Assets

#### Comment 

</details>
